pkg/sql/logictest/REPOSITORIES.bzl

```diff
@@ -7,17 +7,11 @@ CONFIG_DARWIN_AMD64 = "darwin-10.9-amd64"
 CONFIG_DARWIN_ARM64 = "darwin-11.0-arm64"
 
 _CONFIGS = [
-    ("24.3.12", [
```

Comment:
🤔 should the 24.3 configs be removed?

cc @rail

Reply:
We will remove those when 24.3 is EOL.